packages/paste-website/src/components/site-wrapper/SiteBody.tsx

```diff
@@ -32,7 +32,7 @@ export const SiteBody: React.FC = ({children}) => {
       <StyledSiteBody>
         {breakpointIndex === undefined || breakpointIndex > 1 ? <Sidebar /> : null}
         <Box flex="1" minWidth="size0">
-          <main>
+          <main id="content">
```

Comment:
Can we move this ID to a constants file and also make it less generic? I think it clashes with our modal `__console_patch` hacky prop lol.
Something like `paste-docs-content-area`

Reply:
Good call. I'll make those changes.